package.json

```diff
@@ -8,34 +8,48 @@
   "bugs": "https://github.com/alakarteio/k-redux-factory/issues",
   "scripts": {
     "lint:js": "eslint --ext js,jsx src",
-    "lint": "npm-run-all --parallel lint:*",
+    "lint": "run-p lint:*",
     "cmd:build": "microbundle --external all --sourcemap false -f es,cjs -i ",
-    "build:core": "yarn cmd:build src/index.js",
-    "build:helper": "yarn cmd:build src/helpers.js -o helpers/index.js",
-    "build": "npm-run-all --parallel build:*",
+    "build:core": "run-s \"cmd:build src/index.js\"",
+    "build:helper": "run-s \"cmd:build src/helpers.js -o helpers/index.js\"",
+    "build": "run-p build:*",
     "test": "jest",
     "coveralls": "jest --coverage --collectCoverageFrom=src/**/*.{js,jsx} && cat ./coverage/lcov.info | coveralls",
-    "ci": "npm-run-all --parallel coveralls lint"
+    "ci": "run-p coveralls lint"
   },
   "devDependencies": {
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.0",
     "babel-eslint": "^8.2.1",
     "babel-jest": "^23.0.1",
     "babel-plugin-external-helpers": "^6.22.0",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
```

Comment:
info Reasons this module exists
   - "babel-preset-env" depends on it
   - Hoisted from "babel-preset-env#babel-plugin-transform-es2015-modules-commonjs"

Reply:
> "babel-preset-env" depends on it

I don't know why it doesn't work without explicitly add it